Image Scrapper

```diff
@@ -0,0 +1,34 @@
+from bs4 import BeautifulSoup
+import requests
+import urllib.request
+
+from google.colab import drive
+drive.mount('/content/drive')
+
+import pandas as pd
+df = pd.read_csv('/content/drive/MyDrive/Ralph_Lauren_Retail_Feed.csv')
+print(df.shape)
+df.head()
+
+for i in range(len(df)):
```

Comment:
To fit the pattern we've been using everywhere else can you wrap this in a `main()` function and check the name guard at the bottom? `if __name__ == '__main__'`